subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/processing/MultipleOriginProcessor.java

```diff
@@ -27,12 +29,12 @@
  */
```

Comment:
it's not in this change, but the comment on this class describes a single origin processor rather a multiple origin processor.